config.mk

```diff
@@ -4,7 +4,7 @@ VERSION = 0
 # customize below to fit your system
 
 # paths
-PREFIX = /usr/local
+PREFIX = /usr
```

Comment:
This is not really needed as it will be override at build time anyways

Reply:
Alright